osquery/devtools/shell.cpp

```diff
@@ -670,9 +668,8 @@ static void output_csv(struct callback_data *p, const char *z, int bSep) {
 */
 static void interrupt_handler(int signal) {
   if (signal == SIGINT) {
-    osquery::Initializer::requestShutdown(130);
```

Comment:
Note, that the previous introduction of ^C in the shell is now (again) removed. The shell does not really support an interrupt nor does that experience mimic normal shell REPLs.